src/core/Akka.Remote.Tests/RemoteRouterSpec.cs

```diff
@@ -110,7 +110,7 @@ public void RemoteRouter_must_deploy_its_children_on_remote_host_driven_by_confi
             for (var i = 0; i < 5; i++)
             {
                 router.Tell("", probe.Ref);
-                var expected = probe.ExpectMsg<ActorRef>();
+                var expected = probe.ExpectMsg<ActorRef>(TimeSpan.FromSeconds(5));
```

Comment:
I think we should use this TestKit helper:

```csharp
        /// <summary>
        /// If <paramref name="timeout"/> is defined it is returned; otherwise
        /// the config value "akka.test.single-expect-default" is returned.
        /// </summary>
        public TimeSpan GetTimeoutOrDefault(TimeSpan? timeout)
```

`GetTimeoutOrDefault(null)` gives you the default expect timeout by config
Just to avoid magic constants IMO.